packages/x-tailwindcss/src/x-tailwind-plugin/components/button/default.ts

```diff
@@ -0,0 +1,56 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the default styles for component `button`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the component.
+ */
+// eslint-disable-next-line  @typescript-eslint/explicit-function-return-type
+export function buttonDefault({ theme }: TailwindHelpers) {
+  return {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    alignContent: 'center',
+    flexFlow: 'row nowrap',
+    boxSizing: 'border-box',
+    minHeight: theme('spacing.48'),
+    gap: theme('spacing.8'),
+    paddingInlineStart: theme('spacing.16'),
+    paddingInlineEnd: theme('spacing.16'),
+    borderRadius: theme('borderRadius.none'),
+
+    borderStyle: 'solid',
+    borderWidth: theme('borderWidth.1'),
+
+    borderColor: theme('colors.neutral.50'),
+    backgroundColor: theme('colors.neutral.50'),
+    color: theme('colors.neutral.0'),
+
+    fontFamily: theme('fontFamily.primary'),
+    fontSize: theme('fontSize.md'),
+    fontWeight: theme('fontWeight.bold'),
+    letterSpacing: theme('letterSpacing.md'),
+    lineHeight: theme('lineHeight.sm'),
+
```

Comment:
I'm wondering if we should add the `cursor: default` as it is specified on figma. I think during the pairing we discussed about it and agreed on writing down every attribute from figma?

Reply:
ok, added

Reply:
@tajespasarela I think you forgot to push the change? at least I can't see it 😅

Reply:
I do see it up:
<img width="803" alt="image" src="https://user-images.githubusercontent.com/5759712/193258793-9cfa6226-3446-4ca4-afda-375443419796.png">